controllers/datafoundationclient.go

```diff
@@ -115,6 +124,25 @@ func (r *dataFoundationClientReconciler) parseSpec(offering *v1alpha1.ManagedFus
 
 func (r *dataFoundationClientReconciler) reconcilePhases() (ctrl.Result, error) {
 	if !r.offering.DeletionTimestamp.IsZero() {
+		found, err := r.hasOCSVolumes()
+		if err != nil {
+			return ctrl.Result{}, err
+		}
+		if found {
+			r.Log.Info("Found consumer PVs using default storageclasses, cannot proceed with uninstallation")
+			return ctrl.Result{Requeue: true, RequeueAfter: 10 * time.Second}, nil
+		}
```

Comment:
```suggestion
		if found, err := r.hasOCSVolumes(); err != nil {
			return ctrl.Result{}, err
		} else if found {
			r.Log.Info("Found consumer PVs using default storageclasses, cannot proceed with uninstallation")
			return ctrl.Result{Requeue: true, RequeueAfter: 10 * time.Second}, nil
		}
```
Better to not pollute scope of variables

Reply:
updated